docs/docs/how-to-guides/vtex-integration/introduction.md

```diff
@@ -0,0 +1,39 @@
+
+## How to integrate FastStore with VTEX
+
+FastStore can be integrated with VTEX to provide a great checkout and post-purchase experience. Follow the steps below to know how.
```

Comment:
```suggestion
FastStore can be integrated with VTEX to provide a great checkout and post-purchase experience. Follow the steps below to know-how.
```

Reply:
know-how is the noun, I believe he wanted the verb+adverb